### PR TITLE
Create Rule “different-levels-of-testing/rule”

### PR DESCRIPTION
**Pull request by leehawkinsssw**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/different-levels-of-testing/rule.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+See [Rules to Better Unit Tests](https://www.ssw.com.au/rules/rules-to-better-unit-tests)
+
+### Acceptance tests (aka "service tests" or "API tests")
+
+The middle layer of the pyramid - variously referred to as acceptance tests, service tests or API tests - increases the scope of tests compared to unit tests and is often (as Cohn refers to it) the "forgotten layer". While there is great benefit to be gained from automating at this level, it is often ignored or overlooked in automation efforts, especially in teams that are [overly-reliant on automated UI tests](URL).
```
Comment (tiagov8) [inline rules/different-levels-of-testing/rule.md:51]: Hey @leehawkinsssw 

Missing the URL here

Comment (leehawkinsssw) [reply rules/different-levels-of-testing/rule.md:51]: Should be https://ssw.com.au/rules/automated-ui-testing-sparingly

**Review by adamcogan — DISMISSED** [review]
@leehawkinsssw lets talk about your use of: endintro

File: rules/different-levels-of-testing/rule.md (hunk 1 above)
Comment (tiagov8) [reply rules/different-levels-of-testing/rule.md:51]: Hey @leehawkinsssw 

You should be able to edit the file in your PR to fix this. Then mark as solved and request a new review

**Comment by tiagov8** [comment]
> @leehawkinsssw lets talk about your use of: endintro

I would have added it on line 21 (before the image)

**Comment by leehawkinsssw** [comment]
> > @leehawkinsssw lets talk about your use of: endintro
> 
> I would have added it on line 21 (before the image)

OK, is there an unwritten rule here that images shouldn't be part of the blurb (i.e. before the endintro tag)? We should mention it in the appropriate spot of the Rules wiki if this is a stylistic preference. @tiagov8 @adamcogan

File: rules/different-levels-of-testing/rule.md (hunk 1 above)
Comment (leehawkinsssw) [reply rules/different-levels-of-testing/rule.md]: Done

**Comment by leehawkinsssw** [comment]
> > @leehawkinsssw lets talk about your use of: endintro
> 
> I would have added it on line 21 (before the image)

Done

**Review by tiagov8 — APPROVED** [review]

**Comment by tiagov8** [comment]
> > > @leehawkinsssw lets talk about your use of: endintro
> > 
> > 
> > I would have added it on line 21 (before the image)
> 
> OK, is there an unwritten rule here that images shouldn't be part of the blurb (i.e. before the endintro tag)? We should mention it in the appropriate spot of the Rules wiki if this is a stylistic preference. @tiagov8 @adamcogan

Done - I added a note to the "endintro" section at https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Create-Rules#manual-creation-in-github

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**